### ISSUE TITLE
Featured apps have superfluous alt text on images

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4584]
Every featured app has 2 images (a screenshot and an icon) and a heading text. Both images have alt text that basically indicate what the featured app is. The heading indicates this as well. The images can be marked up with an empty alt-attribute: alt="". This indicates that the images are decorative (because their functionality is already covered by the heading). Right now the accessible name (how the link is pronounced) is very verbose by mentioning the app name thrice.

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos
